src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -106,20 +85,6 @@ export const AssetActions: React.FC<FoxTabProps> = ({ assetId }) => {
           </TabList>
           <TabPanels>
             <TabPanel textAlign='center' p={6}>
-              <Box mb={6}>
-                <AssetIcon
-                  src={coinbasePayFeatureFlag && isFoxAsset ? coinbaseLogo : asset.icon}
-                  boxSize='16'
-                />
-              </Box>
-              <SkeletonText isLoaded={Boolean(description?.length)} noOfLines={3}>
-                <CText color='gray.500' mb={6}>
-                  {coinbasePayFeatureFlag && isFoxAsset
-                    ? translate('plugins.foxPage.purchaseFox')
-                    : trimmedDescription}
-                </CText>
-              </SkeletonText>
-
```

Comment:
We should keep this node, just remove the coinbase flag left side, see prod vs. this diff

<img width="398" alt="image" src="https://user-images.githubusercontent.com/17035424/181116155-42ff6347-93eb-440a-b4e9-9260d0b8c0be.png">
<img width="404" alt="image" src="https://user-images.githubusercontent.com/17035424/181116229-fe97490d-59b1-4f7c-80ef-3e37b23b3372.png">

Reply:
yeah i got a little heavy handed rambo here i'll bring this back